### PR TITLE
DOC: Add CIFTI usage to documentation

### PR DESCRIPTION
**Pull request by smeisler**
Having trouble getting the code segments to appear as code (usually surrounding with "`" works but I guess .rst is different?)

### CODE REVIEW

**Comment by effigies** [comment]
Use double-backticks. Single backticks in RST are for link-like objects.

**Comment by smeisler** [comment]
Thanks, fixed.

File: docs/usage.rst [hunk 1]
```diff
@@ -30,3 +30,8 @@ Command-Line Arguments
    :prog: fitlins
    :nodefault:
    :nodefaultconst:
+
+A Note about Processing CIFTIs
+======================
+
+FitLins (as of v0.7.0) can process CIFTI outputs (e.g., from fMRIPrep's ``--cifti-output`` flag). For FitLins to load fMRIPrep CIFTI outputs, define ``--space fsLR`` and ``--desc-label ""``. The FitLins statistical map outputs will be saved as ``.dscalar.nii`` images. Figures in the reports will include a surface representation as well as separate images for subcortical regions.
```
Comment (effigies) [inline docs/usage.rst]:
```suggestion
FitLins (as of v0.7.0) can process CIFTI outputs (e.g., from fMRIPrep's ``--cifti-output`` flag).
For FitLins to load fMRIPrep CIFTI outputs, define ``--space fsLR`` and ``--desc-label ""``.
The FitLins statistical map outputs will be saved as ``.dscalar.nii`` images.
Figures in the reports will include a surface representation as well as separate images for subcortical regions.
```

File: docs/usage.rst [hunk 2]
```diff
@@ -30,3 +30,11 @@ Command-Line Arguments
    :prog: fitlins
    :nodefault:
    :nodefaultconst:
+
+A Note about Processing CIFTIs
+======================
```
Comment (effigies) [inline docs/usage.rst]:
```suggestion
A Note about Processing CIFTIs
==============================
```

**Comment by effigies** [comment]
Thanks!

**Pull request merged**